src/storage/tuple.cpp

```diff
@@ -230,14 +230,14 @@ void Tuple::DeserializeFrom(UNUSED_ATTRIBUTE SerializeInput &input,
    * out of Tuple. The memory allocation will be performed when serializing
    * to tuple storage.
    */
-  /*const bool is_inlined = tuple_schema->IsInlined(column_itr);
+  /*const bool is_inlined = tuple_schema_->IsInlined(column_itr);
```

Comment:
I saw this but didn't know why it was commented out. So I just left it there for now.